docs/reference/release-notes/7.3.asciidoc

```diff
@@ -0,0 +1,425 @@
+* Permit API Keys on Basic License {pull}42787[#42787]
+
+Authorization::
+* Add "manage_api_key" cluster privilege {pull}43728[#43728] (issue: {issue}42020[#42020])
+* Use separate BitSet cache in Doc Level Security {pull}43669[#43669] (issue: {issue}30974[#30974])
+* Support builtin privileges in get privileges API {pull}42134[#42134] (issue: {issue}29771[#29771])
+
+CCR::
+* Replicate aliases in cross-cluster replication {pull}41815[#41815] (issue: {issue}41396[#41396])
+
+Cluster Coordination::
+* Add voting-only master node {pull}43410[#43410] (issue: {issue}14340[#14340])
+* Defer reroute when nodes join {pull}42855[#42855]
+* Stop SeedHostsResolver on shutdown {pull}42844[#42844]
+* Log leader and handshake failures by default {pull}42342[#42342] (issue: {issue}42153[#42153])
+
+Data Frame::
+* [ML][Data Frame] add node attr to GET _stats {pull}43842[#43842] (issue: {issue}43743[#43743])
+* [ML][Data Frame] Add deduced mappings to _preview response payload {pull}43742[#43742] (issue: {issue}39250[#39250])
+* Add version and create_time to data frame analytics config {pull}43683[#43683]
+* [ML] Tag destination index with data frame metadata {pull}43567[#43567]
+* [ML][Data Frame] Add support for allow_no_match for endpoints {pull}43490[#43490] (issue: {issue}42766[#42766])
+* [ML][Data Frame] Add version and create_time to transform config {pull}43384[#43384] (issue: {issue}43037[#43037])
+* [ML][Data Frame] have sum map to a double to prevent overflows {pull}43213[#43213]
+* [ML][Data Frame] adds new pipeline field to dest config {pull}43124[#43124] (issue: {issue}43061[#43061])
+* [ML][Data Frame] write a warning audit on bulk index failures {pull}43106[#43106]
+* [ML] [Data Frame] add support for weighted_avg agg {pull}42646[#42646]
+
+Distributed::
+* Improve Close Index Response {pull}39687[#39687] (issue: {issue}33888[#33888])
+
+Engine::
+* Remove sort by primary term when reading soft-deletes {pull}43845[#43845]
+* Refresh translog stats after translog trimming in NoOpEngine {pull}43825[#43825] (issue: {issue}43156[#43156])
+* Expose translog stats in ReadOnlyEngine {pull}43752[#43752]
+* Do not use soft-deletes to resolve indexing strategy {pull}43336[#43336] (issues: {issue}35230[#35230], {issue}42979[#42979], {issue}43202[#43202])
+* Rebuild version map when opening internal engine {pull}43202[#43202] (issues: {issue}40741[#40741], {issue}42979[#42979])
+* Only load FST off heap if we are actually using mmaps for the term dictionary {pull}43158[#43158] (issue: {issue}43150[#43150])
+* Trim translog for closed indices {pull}43156[#43156] (issue: {issue}42445[#42445])
+* Also mmap terms index (`.tip`) files for hybridfs {pull}43150[#43150] (issue: {issue}42838[#42838])
+* Add a merge policy that prunes ID postings for soft-deleted but retained documents {pull}40741[#40741]
+
+Features/Indices APIs::
+* Remove "template" field in IndexTemplateMetaData {pull}42099[#42099] (issue: {issue}38502[#38502])
```

Comment:
It looks like was closed and not merged. I have removed the labels from the issue so it wont show up if you generate these again.